pkg/networkutils/network_test.go

```diff
@@ -466,18 +470,20 @@ func TestSetupHostNetworkWithExcludeSNATCIDRs(t *testing.T) {
 		map[string]map[string][][]string{
 			"nat": {
 				"AWS-SNAT-CHAIN-0": [][]string{
-					{"-d", "10.10.0.0/16", "-m", "comment", "--comment", "AWS SNAT CHAIN", "-j", "RETURN"},
-					{"-d", "10.11.0.0/16", "-m", "comment", "--comment", "AWS SNAT CHAIN", "-j", "RETURN"},
-					{"-d", "10.12.0.0/16", "-m", "comment", "--comment", "AWS SNAT CHAIN EXCLUSION", "-j", "RETURN"},
+					{"-N", "AWS-SNAT-CHAIN-0"},
```

Comment:
Why are there two entries here? Isn't the `-N` enough? Since we have the chain name already

Reply:
Just for more clarity since the rule itself is `-N chainName`, but yeah i can modify it to just be -N

Reply:
I see, it's up to you, I don't have that strong of a preference